docs/content/querying/groupbyquery.md

```diff
@@ -250,6 +250,7 @@ When using the "v2" strategy, the following query context parameters apply:
 |`sortByDimsFirst`|Sort the results first by dimension values and then by timestamp.|
 |`forcePushDownLimit`|When all fields in the orderby are part of the grouping key, the broker will push limit application down to the historical nodes. When the sorting order uses fields that are not in the grouping key, applying this optimization can result in approximate results with unknown accuracy, so this optimization is disabled by default in that case. Enabling this context flag turns on limit push down for limit/orderbys that contain non-grouping key columns.|
 |`forceHashAggregation`|Force to use hash-based aggregation.|
+|`forceSingleThreadedCombine`|Force to use single-threaded combine.|
```

Comment:
Yes, it's needed because streaming aggregation is not possible once the dictionary gets full (https://github.com/druid-io/druid/pull/4704/files#diff-8bb57c81d0c016c3203d4b3af9b741a4R50). Maybe we can add multi-threaded combining with spilling later if it's worthwhile.